.travis.yml

```diff
@@ -15,16 +15,6 @@ matrix:
           env: TOXENV=py2-cover,py3-cover,coverage
         - python: 3.5
           env: TOXENV=docs
-        - python: 2.7
-          env: TOXENV=py27-pyramid12
-        - python: 2.7
-          env: TOXENV=py27-pyramid13
-        - python: 2.7
-          env: TOXENV=py27-pyramid14
-        - python: 2.7
-          env: TOXENV=py27-pyramid15
-        - python: 3.5
-          env: TOXENV=py35-pyramid16
```

Comment:
Is this a sign of things to come with compatibility requiring Pyramid 1.7 or higher?